gluten-ui/src/main/scala/org/apache/spark/sql/execution/ui/GlutenUIUtils.scala

```diff
@@ -16,13 +16,21 @@
  */
 package org.apache.spark.sql.execution.ui
 
+import org.apache.gluten.config.GlutenConfig.GLUTEN_UI_ENABLED
 import org.apache.gluten.events.GlutenEvent
 
 import org.apache.spark.SparkContext
 import org.apache.spark.status.ElementTrackingStore
 
-object GlutenEventUtils {
-  def post(sc: SparkContext, event: GlutenEvent): Unit = {
+object GlutenUIUtils {
+
+  def uiEnabled(sc: SparkContext): Boolean = {
+    sc.ui.isDefined && sc.getConf.getBoolean(
```

Comment:
If gluten_ui is set via the set command, will it still take effect?

Reply:
gluten_ui should be a static config

Reply:
oh, right.